### PR TITLE
Mobile fixes

### PR DESCRIPTION
**Pull request by caterinamennito**
Let's merge Alex's fixes into the mobile branch to have them live

### CODE REVIEW

File: slides/chapter_1_d8fcd4c930027fa4e1c3870c7e7e0ff1.md [hunk 1]
```diff
@@ -122,7 +122,7 @@ key: 78ef256bc0
 
 - Similar to typing in IPython Shell{{3}}
 
-![](https://assets.datacamp.com/production/repositories/288/datasets/e796d683f23a93859149146a91c884e0ff6990ec/script_0_selected.png) {{3}}
+![](https://assets.datacamp.com/production/repositories/288/datasets/5d77655dae41c68303804f0d7fe71b76013a0dfb/mobile_script.png  = 25) {{3}}
```
Comment (TuurDutoit) [inline slides/chapter_1_d8fcd4c930027fa4e1c3870c7e7e0ff1.md:125]: What are these `= <number>`?

File: chapter1.md [hunk 2]
```diff
@@ -178,13 +181,13 @@ Python is perfectly suited to do basic calculations. Apart from addition, subtra
...
 
-The code in the script on the right gives some examples.
+The code in the script gives some examples.
 
 `@instructions`
 Suppose you have $100, which you can invest with a 10% return each year. After one year, it's $100 \times 1.1 = 110$ dollars, and after two years it's $100 \times 1.1 \times 1.1 = 121$. Add code to calculate how much money you end up with after 7 years, and print the result.
 
 `@hint`
-After two years you have $100 \times 1.1 \times 1.1 = 100 \times 1.1^2$. How much do you have after 7 years than? Use `*` and `**`.
+After two years you have `100 * 1.1 * 1.1 = 100  * (1.1 ** 2)`. How much do you have after 7 years then?
```
Comment (TuurDutoit) [inline chapter1.md:190]: Seems like they applied some of the fixes we have in the export script!

**Review by TuurDutoit — APPROVED** [review]

File: slides/chapter_1_d8fcd4c930027fa4e1c3870c7e7e0ff1.md (hunk 1 above)
Comment (caterinamennito) [reply slides/chapter_1_d8fcd4c930027fa4e1c3870c7e7e0ff1.md:125]: the asset size, I guess?

File: chapter1.md (hunk 2 above)
Comment (caterinamennito) [reply chapter1.md:190]: actually, I just realised that we didn't have a patch for this hint, and it's broken in production 😅  . On the other hand, we have a patch for this exercise instructions (to remove MathJax), but it hasn't been fixed here. So it looks like we will either have to fix these few issues ourselves or leave most of the content patches we already have in place

Comment (caterinamennito) [reply chapter1.md:190]: that's probably why Alex fixed this and not all the others instructions/assignment containing MathJax that we are patching

**Pull request merged**